src/openfermion/ops/_symbolic_operator_test.py

```diff
@@ -151,6 +151,17 @@ def test_init_str(self):
         self.assertIn(correct, fermion_op.terms)
         self.assertEqual(fermion_op.terms[correct], -1.0)
 
+    def test_init_long_str(self):
+        fermion_op = DummyOperator1(
+                '-2.0 [0^ 1] +\n\n -1.0[ 2^ 3 ] - []', -1.)
```

Comment:
what is going on with the "\n\n"? Are you just making sure this will work if there is a new line?